src/DynamoCoreWpf/Windows/BrowserWindow.xaml.cs

```diff
@@ -28,12 +18,22 @@ public BrowserWindow(Uri location)
         private void Window_Loaded(object sender, RoutedEventArgs e)
         {
             // hide loading grid when navigation ready
-            this.Browser.Navigated += (s, a) =>
+            Browser.Navigated += (s, a) =>
             {
-                this.LoadingGrid.Visibility = Visibility.Collapsed;
-                this.Browser.Visibility = Visibility.Visible;
+                LoadingTextBlock.Visibility = Visibility.Collapsed;
+                Browser.Visibility = Visibility.Visible;
+
+                var localPath = ((a.Uri == null) ? string.Empty : a.Uri.LocalPath);
+                if (localPath.ToLowerInvariant().Equals("/register"))
```

Comment:
Yes, exactly. But I don't suppose that will change otherwise products around the world (with older `AdWebServices`) will break, too :)